mutations.py

```diff
@@ -0,0 +1,54 @@
+import random
+import os
+
+
+class BaseMutation:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def mutate(input_data: bytes) -> bytes:
```

Comment:
Use the same conventions in all your code. If you use type syntax here, you should you it in fuzzing.py as well

Reply:
Using type syntax is indeed important